src/platform/user/authentication/utilities.js

```diff
@@ -4,10 +4,15 @@ import * as Sentry from '@sentry/browser';
 import recordEvent from '../../monitoring/record-event';
 import environment from '../../utilities/environment';
 
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
 export const authnSettings = {
   RETURN_URL: 'authReturnUrl',
 };
 
+export const ssoe = state => toggleValues(state)[FEATURE_FLAG_NAMES.ssoe];
```

Comment:
This should follow redux conventions and go in a selectors file. 

I would say either`src/platform/user/selectors.js` is appropriate (although that seems to be more tied to the idea of an authenticated user), or creating a new file under `src/platform/user/authentication/selectors.js`